packages/webpack-loader/src/webpackLoader.test.ts

```diff
@@ -269,6 +269,18 @@ describe('webpackLoader', () => {
     },
   });
 
+  // Asserts that "webpackResolveOptions" are handled properly
+  testFixture('webpack-resolve-options', {
+    loaderOptions: {
+      webpackResolveOptions: { conditionNames: ['source', 'require'] },
+    },
+    webpackConfig: {
+      resolve: {
+        conditionNames: ['source', 'import'],
+      },
+    },
```

Comment:
```suggestion
    loaderOptions: {
      webpackResolveOptions: {
        extensions: ['.ts', '.jsx'],
      },
    },
```

Let's make the assertion real as we don't have `package.json` files in this fixture to test against.